src/main/java/com/moManager/data/Budget.java

```diff
@@ -0,0 +1,41 @@
+package com.moManager.data;
+
+import org.hibernate.type.SetType;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+/**
+ * Created by Carlos on 8/20/2017.
+ */
+public class Budget {
+    private @Id @GeneratedValue Long id;
+    private String type;
+    private Long amount;
+    private String date;
```

Comment:
Don't use a string for date, use the Date class in Java. (If not the user can add `cake` as a date)